src/main/kotlin/racingcar/domain/car/CarList.kt

```diff
@@ -1,16 +1,26 @@
 package racingcar.domain.car
 
-import racingcar.domain.record.RacingRecord
-
 data class CarList(
     val cars: List<Car>
 ) {
-    fun race(tryCount: Int) = RacingRecord(cars.map { car -> car.race(tryCount) })
+    fun race(tryCount: Int) {
+        cars.forEach { car -> car.race(tryCount) }
+    }
+
+    fun getWinner(): CarList {
+        val maxPosition = cars.maxOf { it.position }
+        return CarList(cars.filter { it.position == maxPosition })
+    }
+
+    override fun toString(): String {
+        return cars.joinToString(", ") { it.name }
```

Comment:
```suggestion
        return cars.joinToString { it.name }
```
joinToString은 내부적으로 기본 separator를 ", "를 사용하고 있습니다.

Reply:
separator 기본값을 몰랐네요..!
감사합니다!